azure_monitor/src/azure_monitor/exporter.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+class BaseExporter(object):
+    def __init__(self):
+        self._telemetry_processors = []
+
+    def add_telemetry_processor(self, processor):
+        """Adds telemetry processor to the collection. Telemetry processors
+        will be called one by one before telemetry item is pushed for sending
```

Comment:
Nit: The first line of the comment should be a short description of the function. If there are more details, add a blank line and fill in the rest underneath that line. See [pep8 for docstrings](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).